lib/cipherstash/index/bloom_filter.rb

```diff
@@ -60,20 +60,13 @@ def initialize(key, opts = {})
         end
       end
 
-      # Adds the given term to the bloom filter and returns the filter instance.
+      # Adds the given terms to the bloom filter and returns the filter instance.
       #
-      # @param term [String] the term to add
+      # @param term [Array<String>] the terms to add
       #
       # @return [CipherStash::Index::BloomFilter]
-      def add(term)
-        hash = OpenSSL::HMAC.digest("SHA256", @key, term)
-
-        (0..@filter_term_bits-1).map do |slice_idx|
-          slice = hash[2*slice_idx..2*slice_idx+1]
-          bit_position = slice.unpack("S<").first % @filter_size
-          @bits.add(bit_position)
-        end
-
+      def add(terms)
+        terms.each { |term| add_single_term(term) }
```

Comment:
Pro tip in ruby you can call `Array(x)` to guarantee that you have an array (like `List.wrap` in Elixir).

Reply:
Updated: https://github.com/cipherstash/ruby-client/pull/13/files/b2a4c53cc9ca1097a78421fd1441968e8c4dd03f..8e5d65a209110d745517a8cc72d751240c321451